src/app/components/activities-nearby-card/activities-nearby-card.component.scss

```diff
@@ -0,0 +1,178 @@
+@import url('https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap');
```

Comment:
No need to import the URL, since the link to google fonts is in the file, index.html. 

It loads Roboto Flex and Yeseva One.